blocks/v2-testimonial/v2-testimonial.css

```diff
@@ -0,0 +1,174 @@
+/* stylelint-disable selector-class-pattern */
+
+.v2-testimonial {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.v2-testimonial__blockquote-column {
+  background-color: #2b2b2b;
+  padding: 48px 16px;
+  color: var(--c-primary-white);
+  font-family: var(--ff-body);
+  font-size: 14px;
+  font-style: normal;
```

Comment:
You can remove font-style and font-weight

Reply:
Updated